.github/workflows/build.yml

```diff
@@ -5,6 +5,9 @@ on: [push, pull_request]
 # The below variables reduce repetitions across similar targets
 env:
   REMOVE_BUNDLED_BOOST : rm -rf /usr/local/share/boost
+  APT_PURGE_DEFAULT_BOOST : |
```

Comment:
This isn't reused so no need to put this up here.

Reply:
Removed.